app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -539,9 +548,26 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
             is Command.GenerateWebViewPreviewImage -> generateWebViewPreviewImage()
             is Command.LaunchTabSwitcher -> launchTabSwitcher()
             is Command.ShowErrorWithAction -> showErrorSnackbar(it)
+            is Command.OpenSettings -> openDefaultBrowserSettings()
+            is Command.OpenDialog -> openDefaultBrowserDialog(it.url)
         }
     }
 
+    private fun openDefaultBrowserDialog(url: String) {
+        instructionsCard?.show()
+        defaultCard?.show()
+        defaultCard?.animate()?.alpha(1f)?.setDuration(100)?.start()
```

Comment:
Consider using a const for `100` so it can be reused when hiding the card as well.